.github/actions/benchmark/action.yml

```diff
@@ -146,15 +177,22 @@ runs:
   - name: Run only on dev branch
     if: github.ref_name == 'dev'
     run: |
-      mkdir -p $HOME/ubench-results-dev
+      target_dir=$HOME/ubench-results-dev
+      mkdir -p ${target_dir}
 
-      for machine in microvm hyperlight; do
-        for benchmark in boot_time cold_start warm_start warm_start_vmm; do
```

Comment:
We had a hard-coded list of what benchmarks to copy results when running the benchmarks in `dev`, and I'd forgotten to add `round-trip-latency` to the list. Now we are just re-using the input.